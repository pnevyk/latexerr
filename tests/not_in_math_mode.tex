\documentclass{article}

\begin{document}
_
\end{document}
