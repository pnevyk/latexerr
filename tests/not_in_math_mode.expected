File: not_in_math_mode.tex

Error on line 4: String _ is valid only in math mode.
